app/services/albums.js

```diff
@@ -0,0 +1,28 @@
+const request = require('request-promise');
+const errors = require('errors.js');
+const logger = require('/logger/index.js');
+
+const options = {
+  uri: process.env.ALBUMS_API_URL,
+  headers: { 'User-Agent': 'Request-Promise' },
+  json: true
+};
+
+const get = data => {
+  logger.info(`GET ${data}`);
+  request(data).catch(error => {
+    errors.defaultError(error);
+  });
+};
+
+exports.albums = () => {
+  const albumOptions = { ...options };
+  albumOptions.uri = `${options.uri}/albums`;
+  return get(albumOptions);
```

Comment:
Instead of mutate the albums options you can do this, `get({ ...albumOptions, uri: newUri })`